contracts/test/fixture/_pool_tilt.js

```diff
@@ -0,0 +1,142 @@
+const hre = require("hardhat");
+const { ethers, waffle } = hre;
+const { BigNumber } = ethers;
+const { impersonateAndFundContract, mintWETH } = require("./_fixture");
+const addresses = require("../../utils/addresses");
+
+const { ousdUnits } = require("../helpers");
+
+const { factoryCreatePool } = require("./_pools");
+
+const attackerAddress = "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045";
+let sAttacker;
+let pool;
+
+async function getPoolConfiguration(fixture, pool) {
+  const { rEthBPT, sfrxETHwstETHrEthBPT } = fixture;
+
+  if (!!rEthBPT && pool.address == rEthBPT.address) {
```

Comment:
Could use the `?.` operator here like `pool.address == rEthBPT?.address`.

But the only thing that's dynamically fetched here seems to be the `balancerPoolId`, which doesn't change either. All the addresses fetched using `fixture` object are constants as well. So, probably can just make these configuration a global static JSON object?

```
    const poolConfigs = {
        [rEthBPT.address]: { ... },
        [sfrxETHwstETHrEthBPT.address]: { ... }
    }

    function getPoolConfig(pool) {
        return poolConfigs[pool.address];
    }
```

Reply:
There is also a pool and BPT token contracts being populated here, example:
```
pool: sfrxETHwstETHrEthBPT,
bptToken: sfrxETHwstETHrEthBPT
```

and while you are right that all these could be re-created outside of the fixture scope from the global addresses / ABIs I think it is more correct to use the ones created by the fixtures. For the following reasons: 
- re-creating contracts seems wasteful if the fixture has already created them
- fixture could potentially add special functionality by replacing code in the contracts, and it seems more correct to use the same object instances as created by the fixture

Reply:
Agreed, let's leave it as it is for now